department-of-veterans-affairs/vets-website
# BDD: use `va-date` in wizard

## Description

Replacing the `Date` React component with a `va-date` web component withIn the 526 wizard. This is used to determine if the service member is eligible for BDD.

Test it as follows:
- Go to [Form 526's wizard](http://localhost:3001/disability/file-disability-claim-form-21-526ez/start) (local dev url & no need to log in)
- Choose "Yes" for active duty
- Enter a past date
- An error message will appear immediately after entering a valid date (this is broken on staging?!)

## Original issue(s)

https://github.com/department-of-veterans-affairs/va.gov-team/issues/40958
Related: department-of-veterans-affairs/vets-design-system-documentation#877

## Testing done

Updated Cypress e2e tests - it's failing because it won't type into the year input. I've notified the Design System Team.

## Screenshots

No visual change

## Acceptance criteria
- [x] Replace React `Date` component with `va-date`
- [x] Update e2e tests (pending error preventing 

## Definition of done
